_config.yml

```diff
@@ -17,11 +17,13 @@ title: MSSE Bootcamp Blog
 email: msse@berkeley.edu
 description: >- # this means to ignore newlines until "baseurl:"
   A website for the MSSE Bootcamp
-repository: 
-baseurl: "" # the subpath of your site, e.g. /blog
+repository: blog-alpha-chi-omega
+
+
+baseurl: "/blog-alpha-chi-omega" # the subpath of your site, e.g. /blog
 url:  # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: 
-github_username:  
+github_username:  charisliao
```

Comment:
Everyone should not add a git username here. It will be different for everyone and when you try to merge, there will be a conflict. Remove this and make a new commit.